Source/JavaScriptCore/JavaScriptCore.xcodeproj/project.pbxproj

```diff
@@ -2507,7 +2503,6 @@
 			dstPath = "";
 			dstSubfolderSpec = 16;
 			files = (
-				DDB04F41278E569A008D3678 /* libWTF.a in Product Dependencies */,
```

Comment:
This dependency exists because the "Generate Unified Sources" phase uses a script from WTF. I think we _could_ get rid of it, but we would need to add
```
$(BUILT_PRODUCTS_DIR)/usr/local/include/wtf/Scripts/generate-unified-source-bundles.rb
```
to the list of inputs to Generate Unified Sources.

Reply:
I will do that.